tests/scripts/thread-cert/Cert_6_1_07_RouterAttachLinkQuality.py

```diff
@@ -81,6 +87,39 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual(self.nodes[ED].get_state(), 'child')
 
+        self.collect_ipaddrs()
+        addrs = self.nodes[ED].get_addrs()
+        for addr in addrs:
+            self.assertTrue(self.nodes[ROUTER1].ping(addr))
+
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        ROUTER_1 = pv.vars['ROUTER_1']
+        ED = pv.vars['ED']
+        _router1_pkts = pkts.filter_wpan_src64(ROUTER_1)
+        _ed_pkts = pkts.filter_wpan_src64(ED)
+
```

Comment:
nit: use one filter if all steps are checked in a timely manner.

Reply:
@FiveDimensions , is this comment resolved?

Reply:
resolved in commit 5